community-stabilization-fund-api/src/modules/layout/components/UserLayout/ChecklistConfigSection.tsx

```diff
@@ -12,15 +12,39 @@ interface ChecklistConfigSectionProps {
 
 const ChecklistConfigSection = ({
   handleOpen,
-  handleChange
+  handleChange,
 }: ChecklistConfigSectionProps) => {
-  const items = [BAG_LABEL_TYPES.DYMO_LABELS.OPTION_ONE, BAG_LABEL_TYPES.SHEET_LABELS.OPTION_ONE];
+  const items = [
+    BAG_LABEL_TYPES.DYMO_LABELS.OPTION_ONE,
+    BAG_LABEL_TYPES.SHEET_LABELS.OPTION_ONE,
+  ];
 
   return (
     <Column>
-      <Button kind={'primary'} onClick={() => handleOpen('checklistRulesModal')}>
+      <Button
+        kind={'primary'}
+        onClick={() => handleOpen('checklistRulesModal')}
+      >
         Configure Package Item Rules
       </Button>
+      <Button
+        kind={'primary'}
+        onClick={() => handleOpen('addPackageItemModal')}
+      >
+        Add Package Item
+      </Button>
+      <Button
+        kind={'primary'}
+        onClick={() => handleOpen('addPackageGroupModal')}
+      >
+        Add Package Group
+      </Button>
+      <Button
+        kind={'primary'}
+        onClick={() => handleOpen('addPackageGroupItemModal')}
+      >
+        Add Package Group Item
+      </Button>
```

Comment:
You shouldn't need 3 different modals. Make the flow happen in one or two max. PackageGroupItem should be a reference field the existing tables.

One flow: 

Modal -> createPackageGroup -> nextStepButton -> addPackageItemToGroupButton -> createBulkPackageGroupItems/createMany 

Two flow

Modal -> createPackageGroup -> nextStepButton -> addPackageItemToGroupButton -> createBulkPackageGroupItems/createMany 

Modal -> createPackageItem -> nextStepButton -> addPackageItemToGroupButton -> createBulkPackageGroupItems/createMany